easy_my_coop_website/controllers/main.py

```diff
@@ -450,6 +470,10 @@ def share_subscription(self, **kwargs):
 
         values["share_product_id"] = self.get_selected_share(kwargs).id
 
+        values["address"] = kwargs.get("street_name") + ", " + kwargs.get("house_number")
```

Comment:
On this line, I like the use of `get`, that prevent raising error if the key does not exists in the dictionary. But the default value returned if the key is not found in the dict is `None` (https://docs.python.org/3/library/stdtypes.html#dict.get).

This, will lead to a `TypeError` because `None` cannot be concatenated with `str`. So if the goal is to prevent raising error if the key is not present in dict write something like:
```python
values["address"] = "{street}, {number}".format(
    street = kwargs.get("street_name", default=""),
    number = kwargs.get("house_number", default="")
)
```
Note the use of format that is more readable IMO than concatenate with `+`.

Thanks for your work. :)